packages/ckeditor5-widget/src/widgetresize/sizeview.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module widget/widgetresize/sizeview
+ */
+
+import View from '@ckeditor/ckeditor5-ui/src/view';
+
+/**
+ * A view displaying the proposed new element size during the resizing.
+ *
+ * @extends {module:ui/view~View}
+ */
+
+export default class SizeView extends View {
+	constructor() {
+		/**
+		 * The position of the view defined based on the host size and active handle position.
+		 *
+		 * @private
+		 * @observable
+		 * @member {String} #viewPosition
+		 */
+
+		super();
+
+		const bind = this.bindTemplate;
+
+		this.setTemplate( {
+			tag: 'div',
+			attributes: {
+				class: [
+					'ck',
+					'ck-size-view',
+					bind.to( 'viewPosition', value => value ? `ck-orientation-${ value }` : '' )
+				],
+				style: {
+					display: bind.if( 'isVisible', 'none', visible => !visible )
+				}
+			},
+			children: [ {
+				text: bind.to( 'label' )
```

Comment:
Missing `#label` docs. This also looks like an internal state for template binding purposes, so let's make it @Protected and `#_label`.

Reply:
It is not used in tests or outside the class actually, so I made it @Private.